src/brevitas_examples/llm/llm_args.py

```diff
@@ -490,7 +490,7 @@ def validate(args: Namespace, extra_args: Optional[List[str]] = None) -> None:
             assert args.weight_quant_rescaling_init > 0, \
                 'Error: weight_quant_rescaling_init must be positive.'
         if (int(args.gptq) + int(args.gpfq) + int(args.qronos)) > 1:
-            warn("GPTQ, GPFQ, and/or Qronos are enabled together.")
+            assert False, "GPTQ, GPFQ, and/or Qronos are enabled together."
```

Comment:
I'm fine being such a small change. Nit:
```suggestion
            assert (int(args.gptq) + int(args.gpfq) + int(args.qronos)) <= 1, "GPTQ, GPFQ, and/or Qronos cannot be enabled together."
```